internal/monitors/pyrunner/linux.go

```diff
@@ -21,7 +23,11 @@ func procAttrs() *syscall.SysProcAttr {
 }
 
 func pythonBinaryExecutable() string {
-	return filepath.Join(os.Getenv(constants.BundleDirEnvVar), "lib64", "ld-linux-x86-64.so.2")
+	if strings.Compare(runtime.GOARCH, "arm64") == 0 {
```

Comment:
The idiomatic way of comparing strings in Go is with just regular `==`, none of that Java confusion.

Reply:
done